src/Voron/Data/Tables/Table.cs

```diff
@@ -2225,6 +2225,26 @@ public TableReport GetReport(bool includeDetails, StorageReportGenerator generat
             return report;
         }
 
+        public bool? TableValueIsCompressed(Slice key, out bool? isLargeValue)
```

Comment:
Where is this used?

Reply:
Renamed and moved to the `ForTestingPurposesOnly()`.